t/plugin/request-id2.t

```diff
@@ -70,39 +70,6 @@ passed
 
 
 
-=== TEST 2: add plugin with algorithm range_id
```

Comment:
Why need remove this test?  Maybe we should add a test to hit route, it can cover this bug. eg:
```
=== TEST 3: hit
--- request
GET /opentracing
```

Reply:
Agree with @jiangfucheng